docker-stack.yml

```
@@ -49,6 +63,7 @@ services:
       DATABASE_URL: postgres://cfpb@postgres/cfgov
       DJANGO_ADMIN_USERNAME: admin
       ES_HOST: elasticsearch
+      ES7_HOST: http://elasticsearch7:9200
```

This needs to be like this now, doesn't it?
```suggestion
      ES_HOST: elasticsearch2
      ES7_HOST: http://elasticsearch:9200
```

---

It doesn't but it might make more sense to do this swap. 

The docker stack is still deploying elasticsearch 2.3.5 as a container named elasticsearch, and I've just added elasticsearch7. This is not consistent with how we run them locally via docker-compose and we can make that change.

---

Yeah, I'd like to see this so it's consistent with the conventions in the `docker-compose.yml` file.

---

Done!